File: initFileList.java
public class InitFileList {
    public static void main(String[] args) {
        System.out.println("InitFileList started");

        FnsEngine fnsUL = new FnsEngine(settings.proxies, settings.headers, settings.ulcert,
                settings.ulkey, settings.url, settings.ulstart_dir, settings.ex_dir);
        System.out.println("EGRUL engine init done");

        FnsEngine fnsIP = new FnsEngine(settings.proxies, settings.headers, settings.ipcert,
                settings.ipkey, settings.url, settings.ipstart_dir, settings.ex_dir);
        System.out.println("EGRIP engine init done");

        fnsUL.fileListToFile("EGRUL.txt");
        System.out.println("EGRUL.txt done");

        fnsIP.fileListToFile("EGRIP.txt");
        System.out.println("EGRIP.txt done");
    }
}

File: initFileList.py
import FnsEnginie
import settings

if __name__ == '__main__':
    print ('InitDileList started')
    fnsUL = FnsEnginie.FnsDownload(proxies = settings.proxies, headers = settings.headers,
            cert_str = settings.ulcert, key_srt = settings.ulkey, url = settings.url,
            start_dir = settings.ulstart_dir, ex_dir = settings.ex_dir)
    print('EGRUL enginie init done')
    fnsIP = FnsEnginie.FnsDownload(proxies = settings.proxies, headers = settings.headers,
            cert_str = settings.ipcert, key_srt = settings.ipkey, url = settings.url,
            start_dir = settings.ipstart_dir, ex_dir = settings.ex_dir)
    print('EGRIP enginie init done')
    fnsUL.file_list_to_file('EGRUL.txt')
    print('EGRUL.txt done')
    fnsIP.file_list_to_file('EGRIP.txt')
    print('EGRIP.txt done')

